sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -386,7 +386,19 @@ def _management_request(self, mgmt_msg, op_type):
                 mgmt_client.open()
                 while not mgmt_client.client_ready():
                     time.sleep(0.05)
-                mgmt_msg.application_properties["security_token"] = mgmt_auth.get_token()
+                access_token = mgmt_auth.get_token()
+
+                if not access_token:
+                    _LOGGER.info("Management client received an access token object")
```

Comment:
Shouldn't this be something like "Management client received an **empty** access token object"?